src/axom/inlet/docs/sphinx/verification.rst

```diff
@@ -0,0 +1,55 @@
+.. _inlet_verification_page_label:
+
+############
+Verification
+############
+
+Before input file data can be accessed, it must first be verified by calling the ``verify()``
+method of the top-level ``Inlet`` object. This will return a ``bool`` indicating whether the
+provided input conformed to the schema, and specific violations of the schema are logged via
+SLIC.
+
+This section describes the verification rules that apply to each possible element of the Inlet
+hierarchy, namely, ``Container``, ``Field``, and ``Function``.
+
+Container Verification
+----------------------
+
+If a ``Container`` is marked as required (via the ``required()``) method, then the ``Container`` must be nontrivial,
```

Comment:
not sure "nontrivial" brings anything to this sentence. Also the comma in "input, or contains a" is unneeded